app/models/notifications/mentioned_in_comment.rb

```diff
@@ -11,7 +11,7 @@ def deleted_translation_key
     end
 
     def self.filter_mentions(mentions, mentionable, _recipient_user_ids)
-      mentions.joins(:person).merge(Person.allowed_to_be_mentioned_in_a_comment_to(mentionable.parent))
+      mentions.includes(:person).merge(Person.allowed_to_be_mentioned_in_a_comment_to(mentionable.parent)).distinct
```

Comment:
`joins` simply adds a `INNER JOIN` for the query, `includes` includes the person in the result. With `joins` a followed `mention.person` queries the person again, while with `includes` it is already included.